module/lezioni.py

```diff
@@ -67,24 +83,24 @@ def lezioni_cmd(bot, update, args):
         daylist = list(calendar.day_name)
         daylist = [str(x).lower().replace('ì', 'i') for x in daylist]
         #Clear arguments - Trasform all to lower case utf-8 (ì) - Remove word 'anno' and len<2
-        args = [str(x).lower() for x in args if len(x) > 2]
+        args = [str(x).lower().replace('ì', 'i') for x in args if len(x) > 2]
```

Comment:
may we could choose to replace `i` to `ì`, but never mind xD

Reply:
Replacing i to ì for every arg would be a mess 😅
Imagine if the user inputs /lezioni programmazione:
programmazione will be changed to programmazìone, which will then output "Nessun risultato trovato :(" because there is no teaching with that name.

Also, since 'ì' in days were already replaced with i, I've just tried to stick to the previous implementation so that minimum changes were made

Reply:
that's right